inventory/views.py

```diff
@@ -1,72 +1,139 @@
+            member = form.cleaned_data["member"]
+            title = form.cleaned_data["title"]
+            description = form.cleaned_data["description"]
+            width = form.cleaned_data["width"]
+            height = form.cleaned_data["height"]
+            length = form.cleaned_data["length"]
+            quantity = form.cleaned_data["quantity"]
+            unit_price = form.cleaned_data["unit_price"]
+            picture = form.cleaned_data["picture"]
+            voltage = form.cleaned_data["voltage"]
+            wattage = form.cleaned_data["wattage"]
+            # item = Item(member=member,title=title,description=description,width=width,height=height,length=length,quantity=quantity,unit_price=unit_price,picture=picture)
+            # item.save()
+            item = ElectricalItem.objects.create(
+                member=member,
+                title=title,
+                description=description,
+                width=width,
+                height=height,
+                length=length,
+                quantity=quantity,
+                unit_price=unit_price,
+                picture=picture,
+                voltage=voltage,
+                wattage=wattage,
+            )
+            item.save()
+            category = Category.objects.get(pk=category_id)
+            category.items.add(item)
+            category.save()
+            return redirect("inventory:index")
+        else:
+            return render(request, "inventory/AddItem.html", {"form": form})
+    else:
+        form = AddElectricalItemForm()
+        return render(request, "inventory/AddItem.html", {"form": form})
+
+
+def edit_item(request, item_id):
     item = Item.objects.get(pk=item_id)
     if request.method == "POST":
-        form = AddItemForm(request.POST,request.FILES,instance=item)
+        form = AddItemForm(request.POST, request.FILES, instance=item)
         if form.is_valid():
             form.save()
             return redirect("inventory:index")
         else:
-            return render(request, 'inventory/EditItem.html', {'form': form})
+            return render(request, "inventory/EditItem.html", {"form": form})
     else:
         form = AddItemForm(instance=item)
-        return render(request, 'inventory/EditItem.html', {'form': form})
+        return render(request, "inventory/EditItem.html", {"form": form})
+
+
+def edit_electrical_item(request, item_id):
+    item = ElectricalItem.objects.get(pk=item_id)
+    if request.method == "POST":
+        form = AddElectricalItemForm(request.POST, request.FILES, instance=item)
+        if form.is_valid():
+            form.save()
+            return redirect("inventory:index")
+        else:
+            return render(request, "inventory/EditItem.html", {"form": form})
+    else:
+        form = AddElectricalItemForm(instance=item)
+        return render(request, "inventory/EditItem.html", {"form": form})
+
 
-def Delete_Item(request, item_id):
+def delete_item(request, item_id):
```

Comment:
there's no authentication on your functions! 
anyone can do these

There's a decorator you can use for authentication!